src/assets/custom/js/websiteHeader.js

```diff
@@ -31,42 +31,59 @@ var websiteHeader = function () {
 
   function handleScrollPosition(latestWindowPosition) {
     var upwardMovement =
-      previousWindowPosition > latestWindowPosition + tolerance;
+      previousWindowPosition > latestWindowPosition + scrollTolerance;
     var downwardMovement =
-      previousWindowPosition < latestWindowPosition - tolerance;
+      previousWindowPosition < latestWindowPosition - scrollTolerance;
 
-    var subMenuOpen = HEADER.classList.contains(HEADER_HAS_OPEN_SUBMENU_CLASS);
+    var subMenuOpen = header.classList.contains(
+      'website-header--has-open-submenu'
+    );
 
-    if (atTheTop(latestWindowPosition) || (downwardMovement && !subMenuOpen)) {
-      restoreNaturalPosition();
+    if (
+      startingPositionIsInView(latestWindowPosition) ||
+      atTheTop(latestWindowPosition) ||
+      upwardMovement
+    ) {
+      revealHeader();
       return;
     }
 
-    if (startingPositionIsInView(latestWindowPosition)) {
-      return;
-    }
+    if (downwardMovement && !websiteHeaderOpen()) {
+      if (subMenuOpen) {
+        closeSubMenu =
+          window.__CODURANCE.websiteNavigation.closeOpenSubMenu ||
```

Comment:
Just curious why an empty function call is needed, would it possible to not call a function at all if websiteNavigation.closeOpenSubMenu is undefined? :)

Reply:
This is now taken care of in this commit: https://github.com/codurance/site/pull/1940/commits/bbea935a1c2a48d0f6f53ee09248e47647187d59

The empty function is to stop the script erroring and breaking if `window.__CODURANCE.websiteNavigation.closeOpenSubMenu` wasn't defined.